<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>481</width>
      <height>931</height>
     </rect>
    </property>
    <property name="title">
     <string>Graphics</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x-t Graphics</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>470</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>y-t Graphics</string>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>420</width>
       <height>420</height>
      </rect>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlotSecond" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>490</y>
       <width>420</width>
       <height>420</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>611</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>X-Y Coordinates</string>
    </property>
    <widget class="QGraphicsView" name="xyCoordinates">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>530</width>
       <height>490</height>
      </rect>
     </property>
     <property name="resizeAnchor">
      <enum>QGraphicsView::NoAnchor</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Game Area</string>
    </property>
    <widget class="QGraphicsView" name="shapeDrawer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>450</width>
       <height>350</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="drawCircle">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawSquare">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Square</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawRect">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Rectangle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBalance">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Balance</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1335</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
